shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/env/enums/ScalingITTypeEnum.java

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.shardingsphere.integration.data.pipeline.env.enums;
 
-public enum ITEnvTypeEnum {
+public enum ScalingITTypeEnum {
     
```

Comment:
`ScalingITTypeEnum` could be `ScalingITEnvTypeEnum`, keep the same as `scaling.it.env.type`